ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini

```
@@ -96,6 +96,11 @@ solr_url = http://%(solr_server)s:{{ inventory_ckan_solr_port }}/solr/{{app_type
 
 #ckan.simple_search = 1
 
+## Redis Settings
+{% if inventory_ckan_redis_password is defined %}
+ckan.redis.url = redis://empty_username:{{ inventory_ckan_redis_password }}@{{ inventory_ckan_redis_host }}:6379/0
```

If it's an empty username, you can just leave it empty :) This also better matches the value for catalog-next.

```suggestion
ckan.redis.url = redis://:{{ inventory_ckan_redis_password }}@{{ inventory_ckan_redis_host }}:6379/0
```

---

You might want to pull the URL out into its own Ansible value. If we end up sharing redis between catalog-next and inventory next, you'll want a different database value (redis://host:6379/1). Are you planning to add a new redis host?